packages/lesswrong/components/posts/PostsListSortDropdown.tsx

```diff
@@ -41,25 +40,20 @@ const PostsListSortDropdown = ({classes, value}:{
 
   return <div className={classes.root}>
     <span className={classes.selectMenu} onClick={e=>setAnchorEl(e.currentTarget)}>
-      {TAG_POSTS_SORT_ORDER_OPTIONS[value]} <ArrowDropDownIcon className={classes.icon}/>
+      {TAG_POSTS_SORT_ORDER_OPTIONS[value].label} <ArrowDropDownIcon className={classes.icon}/>
     </span>
     <Menu
       open={Boolean(anchorEl)}
       anchorEl={anchorEl}
       onClose={()=>setAnchorEl(null)}
     >
-        <QueryLink query={{sortedBy: undefined}} merge className={classes.menuItem}>
-          <MenuItem value={"relevance"} onClick={()=>setAnchorEl(null)}>
-            {TAG_POSTS_SORT_ORDER_OPTIONS["relevance"]}
```

Comment:
this was unnecessary as all the SORT_ORDER_OPTIONS were already in TAG_POSTS_SORT_ORDER_OPTIONS